tests/test_controller.py

```diff
@@ -846,66 +844,23 @@ def test_report_problem_post_success(self):
     def test_series(self):
         # If the work doesn't have a series, a ProblemDetail is returned.
         with self.app.test_request_context('/'):
-            response = self.manager.work_controller.series(
-                self.datasource, self.identifier.type, self.identifier.identifier
-            )
+            response = self.manager.work_controller.series("")
         eq_(404, response.status_code)
         eq_("http://librarysimplified.org/terms/problem/unknown-lane", response.uri)
 
-        # If the work is in a series without other volumes, a feed is
-        # returned containing only that work.
+        # If the work is in a series, a feed is returned.
         self.lp.presentation_edition.series = "Like As If Whatever Mysteries"
-        self.lp.presentation_edition.series_position = 8
         SessionManager.refresh_materialized_views(self._db)
         with self.app.test_request_context('/'):
             response = self.manager.work_controller.series(
-                self.datasource, self.identifier.type, self.identifier.identifier
+                "Like As If Whatever Mysteries"
             )
         eq_(200, response.status_code)
         feed = feedparser.parse(response.data)
         eq_("Like As If Whatever Mysteries", feed['feed']['title'])
         [entry] = feed['entries']
         eq_(self.english_1.title, entry['title'])
 
-        # Remove cache.
```

Comment:
Confirmed! In `TestSeriesLane.test_works_query` I'm making sure that the number and order of queries is what I expect, so here I'm just making sure that the query result is returned.